### ISSUE TITLE
Intro to game

### ISSUE THREAD

**Issue by ediehouston** [jeffaroundtheworld/2024gamecollab#74]

**Comment by ediehouston** [comment]
Do we want her to wake up and maybe open her eyes and look around. I think that might be cool. And then the text on the screen too.

**Comment by kodriq** [comment]
yeah i think so too, but we have to figure out how we would make her open her eyes and stuff, i can look into it a bit

**Comment by ediehouston** [comment]
Yeah, I saw people saying for 3D you can animate it on Blender. So maybe we can try that.

**Comment by ediehouston** [comment]
Or I'm thinking if all we need is for the character to wake up and look around, maybe we could just screen record, then somehow edit the video to make it look like her eyes are opening and add the text then save that video and use a videostreamplayer node in Godot to play it.

**Comment by ediehouston** [comment]
Ok I know using AE there's definitely a way to edit opening eyes, I saw a video on it.

**Comment by jeffaroundtheworld** [comment]
I like the Screen recording of the room you wake up in as a cut scene... with the wakeup eyes - can be done easily in Premiere Pro (which we have).
Maybe a voiceover talking to yourself?
